client-mr/src/main/java/org/apache/hadoop/mapred/SortWriteBufferManager.java

```diff
@@ -113,7 +114,7 @@ public SortWriteBufferManager(
       int numMaps,
       boolean isMemoryShuffleEnabled,
       int sendThreadNum,
-      double sendThreshold) {
+      double sendThreshold, long maxBufferSize) {
```

Comment:
The new parameter should be start with new line.

Reply:
OK